include/ozo/query_builder.h

```diff
@@ -183,6 +183,7 @@ struct get_query_params_impl<query_builder<Ts...>> {
 
 namespace literals {
 
+# pragma GCC diagnostic ignored "-Wpedantic"
```

Comment:
I'm not sure that this is a good choice to ignore the `pedantic` here without any push/pop. All the tests are green for the master. What problem does this solve?

Reply:
I have problems with compilation without this line. Agreed with push/pop. I will try another time in docker

Reply:
If you have a problem building without docker, could you share the compiler version which causes the problem? In a normal case, there is should not be any difference between the docker image building and the host OS building with the same compiler flags.

Reply:
gcc 9.3.0, ok with clang.
I will remove this